src/lib/helpers/test/setupTestEnv.ts

```diff
@@ -2,17 +2,20 @@ import { createStealthClient } from '../..';
 import { ERC5564_CONTRACT, ERC6538_CONTRACT } from '../../..';
```

Comment:
I'm getting the following when `TEST_LOCAL_NODE_RPC_URL` is not defined.
<img width="1043" alt="image" src="https://github.com/ScopeLift/stealth-address-sdk/assets/61768337/51842a3b-a23b-4906-b0ee-838061a76322">

But when I define it as the anvil local node, I get this
<img width="1149" alt="image" src="https://github.com/ScopeLift/stealth-address-sdk/assets/61768337/1ee83dd2-a060-4e9e-a7a4-c718f8111be8">

But when I change the chainId in `.env` to 31337, I get:
<img width="1054" alt="image" src="https://github.com/ScopeLift/stealth-address-sdk/assets/61768337/a1b2c91c-2b6b-49df-81d3-be66adc6dcc9">

Reply:
https://github.com/ScopeLift/stealth-address-sdk/pull/26/commits/e86150d49212370ccb31ddf6b25c575bd3c61c60

i updated the testing scripts and the readme to hopefully read more clearly on how to set up; still needs a bit more work to make it intuitive.